src/app/shared/services/action-state.service.ts

```diff
@@ -118,4 +118,8 @@ export class ActionStateService {
     this.actionsSubjects.forEach(subject => subject.complete())
     this.eventSubscriptions.forEach(eventSubscription => eventSubscription.unsubscribe())
   }
+
+  public executeAction(actionId: string, rundownId: string): void {
+    this.actionService.executeAction(actionId, rundownId).subscribe()
+  }
```

Comment:
This is just a "parse-through" method. 
You should just inject `ActionService` directly into the component that needs to execute the Action